packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -154,32 +156,34 @@ export function TabList<T extends object>(props: TabListProps<T>) {
   }, [tablistRef, state?.selectedItem]);
 
   return (
-    <div className={style({position: 'relative'})}>
+    <div
+      style={props.UNSAFE_style}
+      className={(props.UNSAFE_className || '') + mergeStyles(style({position: 'relative'}), props.styles)}>
```

Comment:
probably need to call getAllowedOverrides for the style as well
```suggestion
      className={(props.UNSAFE_className || '') + style({position: 'relative'}, props.styles)}>
```

Reply:
should do this with the other components as well